### ISSUE TITLE
Return release notes for store version

### ISSUE THREAD

**Issue by Williano** [timtraversy/new_version#38]
1. Please with this package, can I customize the update message?

2. Is there a way to get the release note from the server? 

Thank you very much for this package

**Comment by timtraversy** [comment]
Hello,

To your first question: yes, as shown in the README.md:

```
final status = await newVersion.getVersionStatus();

newVersion.showUpdateDialog(
  context: context, 
  versionStatus: status,
  dialogTitle: 'Custom dialog title',
  dialogText: 'Custom dialog text',
  updateButtonText: 'Custom update button text',
  dismissButtonText: 'Custom dismiss button text',
  dismissAction: () => functionToRunAfterDialogDismissed(),
)
```

You can pass custom title, text, and buttons to this dialog.

To your second question: That's a good point, and it would probably not be too hard to implement. I'm going to edit this issue so it just captures your second question, and leave it open until it's fixed.

Thanks for the report!

**Comment by Williano** [comment]
Great. I will be monitoring it and please let me know if there is anything I can do to help with this feature.

**Comment by timtraversy** [comment]
Changes merged, just published `v0.2.1` of this package.
